miqa/core/rest/exceptions.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import uuid
+
+from rest_framework import status
+from rest_framework.exceptions import APIException
+from rest_framework.response import Response
+from rest_framework.views import exception_handler
+
+logger = logging.getLogger('MIQA')
+
+
+def custom_exception_handler(exc, context):
+    # Call REST framework's default exception handler first,
+    # to get the standard error response.
+    response = exception_handler(exc, context)
+
+    # If an excpetion is unexpected, it will not be an APIException
+    if not isinstance(exc, APIException):
+        exception_identifier = uuid.uuid4()
+        logger.exception(f'Error {exception_identifier}: {exc} / {context}')
```

Comment:
```suggestion
        logger.exception(f'Unexpected REST API error: {exception_identifier}')
```

We should get the exception data for free by virtue of using `logger.exception`, and `context` should also be dumped to Sentry for free.